app.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Yaydoc",
+  "description": "Automatic Documentation Generation and Deployment Project.",
+  "keywords": [
+    "documentation",
+    "fossasia",
+    "node",
+    "sphinx"
+  ],
+  "repository": "https://github.com/fossasia/yaydoc",
+  "env": {
+    "PYPANDOC_PANDOC": {
```

Comment:
@imujjwal96 Heroku takes care of that for us. Try clicking the deploy to heroku button here -> https://github.com/fossasia/open-event-orga-server :)